scss/partials/_columns.scss

```diff
@@ -4,6 +4,11 @@
   padding-left: 0;
   padding-right: 0;
 
+  h3 {
+    @extend .h4;
+    @include vertical-spacing(sm-3, bottom, margin);
+  }
+
```

Comment:
We will most likely need a rule to add vertical-spacing for `p` tags as well. Something like `@include vertical-spacing(md-3, bottom, padding); `

Reply:
I'll add this in for default column. I'm not sure if other pattern of multi column need it as well since DE&I only have list of text. Please let me know if you wanna add other things relating to this that shows on services page.

Reply:
Rockin! Thanks a bunch. I think that captures changes related to services.